wait/http_test.go

```diff
@@ -22,13 +22,15 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
+const nginxDefaultPort = "80/tcp"
+
 // https://github.com/testcontainers/testcontainers-go/issues/183
 func ExampleHTTPStrategy() {
 	// waitForHTTPWithDefaultPort {
 	ctx := context.Background()
 	req := testcontainers.ContainerRequest{
 		Image:        "nginx:latest",
-		ExposedPorts: []string{"80/tcp"},
+		ExposedPorts: []string{nginxDefaultPort},
```

Comment:
this is used in the docs. By doing so, the docs will not provide the context. It could be happening in other places as well.